pkg/lclient/fakeclient.go

```diff
@@ -143,15 +192,14 @@ func (m *mockDockerClient) ContainerRemove(ctx context.Context, containerID stri
 }
 
 func (m *mockDockerClient) ContainerInspect(ctx context.Context, containerID string) (types.ContainerJSON, error) {
-	for _, containerElement := range mockContainerList {
+	for _, containerElement := range mockContainerJSONList {
 		if containerElement.ID == containerID {
 			containerConfig := container.Config{
-				Image:  containerElement.Image,
-				Labels: containerElement.Labels,
+				Image: containerElement.Image,
 			}
 			return types.ContainerJSON{
 				ContainerJSONBase: &types.ContainerJSONBase{
-					HostConfig: &containerElement.HostConfig,
+					HostConfig: containerElement.HostConfig,
 				},
 				Config: &containerConfig,
 			}, nil
```

Comment:
After implementing https://github.com/openshift/odo/pull/3369#discussion_r441570218, these lines can be removed to 
```suggestion
if containerElement.ID == containerID {
			return containerElement, nil
}
```